.ado/publish.yml

```diff
@@ -39,16 +39,9 @@ jobs:
           script: yarn build
 
       - task: CmdLine@2
-        displayName: Beachball Publish (for master)
+        displayName: Beachball Publish
         inputs:
-          script: node ./node_modules/beachball/bin/beachball.js publish -n $(npmAuthToken) --yes -m "applying package updates ***NO_CI***" --bump-deps --access public
-        condition: and(succeeded(), eq(variables['Build.SourceBranchName'], 'master'))
-
-      - task: CmdLine@2
-        displayName: Beachball Publish (for other branches)
-        inputs:
-          script: node ./node_modules/beachball/bin/beachball.js publish --tag v$(Build.SourceBranchName) --branch origin/$(Build.SourceBranchName) -n $(npmAuthToken) -yes -m "applying package updates ***NO_CI***" --bump-deps  --access public
-        condition: and(succeeded(), ne(variables['Build.SourceBranchName'], 'master'))
+          script: node ./node_modules/beachball/bin/beachball.js publish --branch origin/$(Build.SourceBranchName) -n $(npmAuthToken) -yes -m "applying package updates ***NO_CI***" --bump-deps  --access public
```

Comment:
So I _think_ this will make it publish as latest.  The defaultNpmTag is still set to latest in package.json, no?
https://github.com/microsoft/beachball/blob/43eacf695ff6bc378a33233c47db55b1433eb75c/packages/beachball/src/packageManager/packagePublish.ts#L13
I think we may need to update beachball to handle having no tag specified, as it looks like its specifying --tag in the publish command even if tag and defaultNpmTag are not set.

Reply:
The defaultNpm tag is set to latest, which is what we want here. Beachball has been publishing rnpm-plugin-windows without a tag so far, so I think we should be okay without either? Though we'll just see that in 0.60.